internal/status/gatewayclass.go

```diff
@@ -23,3 +25,25 @@ func SetGatewayClassAccepted(gc *gwapiv1.GatewayClass, accepted bool, reason, ms
 	gc.Status.Conditions = MergeConditions(gc.Status.Conditions, computeGatewayClassAcceptedCondition(gc, accepted, reason, msg))
 	return gc
 }
+
+// GetSupportedFeatures returns a list of supported Gateway-API features,
+// based on the running conformance tests suite.
+func GetSupportedFeatures() []gwapiv1.SupportedFeature {
+
+	// TODO(levikobi): This must be in sync with the cSuite supported features.
```

Comment:
would be ideal to define this, https://github.com/envoyproxy/gateway/blob/100d3103363c64a5be480ee0a2d66e585a9882f1/test/conformance/conformance_test.go#L49 & https://github.com/envoyproxy/gateway/blob/100d3103363c64a5be480ee0a2d66e585a9882f1/test/conformance/experimental_conformance_test.go#L98 once and use the same definition in all 3 places

Reply:
Created a new file under `conformance` package with the unified declaration, let me know if that's ok